src/main/java/org/commonwl/view/util/FileUtils.java

```diff
@@ -47,6 +47,34 @@ public static void deleteGitRepository(Git repo) throws IOException {
     }
   }
 
+  /**
+   * Deletes the directory of a temporary git repository. Note that the <code>Git</code> object
+   * contains a repository with a directory, but this directory points to the
+   *
+   * <pre>.git</pre>
+   *
+   * directory. This method will delete the parent of the
+   *
+   * <pre>.git</pre>
+   *
+   * directory, which corresponds to the cloned folder with the source code from git. Since
+   * temporary folders are generated using <code>UUID.randomUUID()</code> instead of the commit hex
+   * digest, if the folder name contains a '-' character it is identified as temporary.
+   *
+   * @param repo Git repository object
+   * @throws IOException if it fails to delete the Git repository directory
+   * @since 1.4.6
+   */
+  public static void deleteTemporaryGitRepository(Git repo) throws IOException {
+    if (repo != null
+        && repo.getRepository() != null
+        && repo.getRepository().getDirectory() != null
+        && repo.getRepository().getDirectory().getParentFile() != null
+        && repo.getRepository().getDirectory().getParentFile().getName().contains("-")) {
+      deleteGitRepository(repo);
```

Comment:
This line isn't covered by the tests either
https://coveralls.io/builds/55575771/source?filename=src%2Fmain%2Fjava%2Forg%2Fcommonwl%2Fview%2Futil%2FFileUtils.java#L74

Reply:
Same here